docs/src/components/DoDontHeaders.tsx

```diff
@@ -0,0 +1,34 @@
+import { flex } from '@/styled-system/patterns'
+import { css } from '@/styled-system/css'
+import {
+  CheckIcon,
+  CloseIcon,
+  WarningTriangleFilledIcon,
+} from '@pluralsight/react/icons'
+
+export function DoHeader() {
+  return (
+    <span className={flex()}>
+      <CheckIcon className={css({ color: 'success.text.200' })} /> Do
+    </span>
+  )
+}
+
+export function CautionHeader() {
+  return (
+    <span className={flex()}>
```

Comment:
If we do this the JSX can render better and be cleaner. What do you think?

```suggestion
    <span className={hstack({ gap: '2' })}>
```